velox/expression/ExprCompiler.cpp

```diff
@@ -228,6 +229,20 @@ ExprPtr getSpecialForm(
     return std::make_shared<CoalesceExpr>(
         type, std::move(compiledChildren), inputsSupportFlatNoNullsFastPath);
   }
+  if (name == kRowConstructor) {
+    static auto rowConstructorFn =
```

Comment:
This code is duplicated in `if (auto concat = dynamic_cast<const core::ConcatTypedExpr*>(expr.get())) {` branch on L384. Consider extracting a helper function to avoid copy-paste.